Source/ACE.DatLoader/FileTypes/SurfaceTexture.cs

```diff
@@ -0,0 +1,95 @@
+using ACE.Common;
+using ACE.DatLoader.Entity;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ACE.DatLoader.FileTypes
+{
+    [DatFileType(DatFileType.SurfaceTexture)]
+    public class SurfaceTexture : FileType
+    {
+        public int Id { get; private set; }
+        public int Unknown { get; private set; }
+        public byte UnknownByte { get; private set; }
+        public int TextureCount { get; private set; }
+        public List<uint> TextureFileIds { get; } = new List<uint>();
+        public List<Texture> Textures { get; } = new List<Texture>();
+
+        public override void Unpack(BinaryReader reader)
+        {
+            Id = reader.ReadInt32();
+            Unknown = reader.ReadInt32();
+            UnknownByte = reader.ReadByte();
+            TextureCount = reader.ReadInt32();
+            if (TextureCount > 0)
+            {
+                TextureFileIds.Add(reader.ReadUInt32());
+                if (TextureCount > 1)
+                    TextureFileIds.Add(reader.ReadUInt32());
+            }
+
+            if (ConfigManager.Config == null)
+                ConfigManager.Initialize(Path.GetFullPath(@"..\..\..\..\..\ACE.Server\bin\x64\Debug\netcoreapp2.0\Config.json")); // weak
```

Comment:
should this be in here?

Reply:
Unfortunately there doesn't seem to be a kosher way to initialize things.   ACE.DatLoader.Tests, the parent caller in this case, has file paths even harder coded.